api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -32,6 +34,16 @@ message AwsRequestSigning {
     AWS_SIGV4A = 1;
   }
 
+  message QueryString {
+    // Optional expiration time for the query string parameters. As query string parameter based requests are replayable, in effect representing
+    // an API call that has already been authenticated, it is recommended to keep this expiration time as short as feasible.
+    // This value will default to 5 seconds and has a maximum value of 3600 seconds (1 hour).
+    google.protobuf.Duration expiration_time = 1 [(validate.rules).duration = {
```

Comment:
In config.cc I now have

  uint16_t expiration_time =
      PROTOBUF_GET_SECONDS_OR_DEFAULT(config.query_string(), expiration_time, 5);

and a test case to validate this. It should allow user to use query_string: {} and it will default to 5 automatically